packages/handlersjs-http/lib/servers/node/node-http-request-response.handler.ts

```diff
@@ -133,7 +133,7 @@ export class NodeHttpRequestResponseHandler extends NodeHttpStreamsHandler {
           case 405: return of({ ...error, body: 'Method Not Allowed' });
           case 500: return of({ ...error, body: 'Internal Server Error' });
           // If no cases match, return a default error message.
-          default: return error.status < 600 && error.status >= 400 ?  of({ ...error, body: 'An Unexpected Error Occured' }) : of({ body: 'An Unexpected Error Occured', headers: error.headers, status: 500 });
+          default: return error.status < 600 && error.status >= 400 ?  of({ ...error, body: 'An Unexpected Error Occurred' }) : of({ body: 'An Unexpected Error Occurred', headers: error.headers, status: 500 });
```

Comment:
Given the separate error handler, this whole block can now be simplified to a catchError that always returns a 500, I think.

Reply:
This is the node-http-request-response.handler.ts as it was. Just fixed a typo here. You want me to change this as well?